File: Solutions/3540/c++/program.cpp
// Copy from Quera

#include <iostream>
using namespace std;

int main()
{
    int n, x, y;
    cin >> n >> x >> y;
    for (int i = 0; i * x <= n; i++)
        if ((n - i * x) % y == 0)
        {
            cout << i << ' ' << (n - i * x) / y << '\n';
            return 0;
        }
    cout << "-1\n";
    return 0;
}

File: Solutions/3540/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt(), x = scanner.nextInt(), y = scanner.nextInt();
        for(int i = 0; i <= n / x; i++)
            if((n - i * x) % y == 0)
            {
                System.out.println(i + " " + (n - i * x) / y);
                return ;
            }
        System.out.println(-1);
    }

}